test/action_buttons_test.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:spotlight_ant/spotlight_ant.dart';
+
+void main() {
+  group('Spotlight Ant and Custom Action button overrides', () {
+    const actions = [
+      SpotlightAntAction.prev,
+      SpotlightAntAction.skip,
+      SpotlightAntAction.next,
+    ];
+    const duration = SpotlightDurationConfig(
+      zoomIn: Duration(milliseconds: 5),
+      zoomOut: Duration(milliseconds: 5),
+      contentFadeIn: Duration.zero,
+      bump: Duration.zero,
+    );
+
+    testWidgets('basic', (WidgetTester tester) async {
```

Comment:
I suggest simulating the issues, `SpotlightShow.of` causing exception, in the unit test.

Reply:
With the changes made, the issue will not appear anymore when action buttons are pressed (and custom handlers are installed).

I added a unit test to verify the new implementation (and ensure skip, next, prev buttons still work). 

Given these action button overrides had no tests before, I assume it would be enough?

Reply:
@evan361425 does this work for you?

Reply:
This looks perfect!!
I have been on a trip during Chinese New Year. Sorry for the delay.

I will publish the fix right now!"

Reply:
新年快樂